CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 * Added deepcopy for metadata in `make_nwbfile_from_metadata`. [PR #545](https://github.com/catalystneuro/neuroconv/pull/545)
 
+* Alter `add_electrodes` behavior when extra property descriptions are present in metadata so that no error is thrown. [PR #558](https://github.com/catalystneuro/neuroconv/pull/558)
```

Comment:
```suggestion
* Alter `add_electrodes` behavior,  no error is thrown if a property is present in the metadata but not in the recoders. This allows to add recording objects that have different properties [PR #558](https://github.com/catalystneuro/neuroconv/pull/558)
```

I think this is is a better description but your is fine as well. Your call if you want to change it.